flight/Modules/GPS/GPS.c

```diff
@@ -234,6 +230,9 @@ static void gpsTask(void *parameters)
 	}
 #endif /* PIOS_GPS_MINIMAL */
 
+	timeOfLastUpdateMs = 0;
```

Comment:
Also now these two variables are only manipulated in the task, they no longer need to be static to the whole file (good improvement).